docs/pages/admin-guides/management/admin/logging.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Logger configuration
+description: Logger configuration for Teleport instance
+---
+
+## Configuration
+
+In the configuration file of the Teleport instance, you can configure the logger's behavior by defining the output
+destination, severity level, and output format.
+
+Sample of `/etc/teleport.yaml` configuration for defining the logger settings.
+
+```yaml
+teleport:
+  log:
+    output: stderr
+    severity: INFO
+    format:
+      output: text
+      extra_fields: [caller, level]
+```
+
+If the output parameter is not defined or set as empty, `stderr` (aliases `err` or `2`) is used by default.
+Other available options for defining the output include `stdout` (aliases `out` or `1`), `syslog` for writing
+to the syslog file, or a filepath for direct writing to a log file destination.
+
+Severity has several levels, which are sorted by decreasing priority (means that if `info` is selected, `warn` and `err` also applied):
+ - `err`, `error` - used for errors that should definitely be noted.
```

Comment:
What does "should definitely be noted" mean?

Reply:
Means that these log messages indicate actions are required to fix them, errors may lead to an unpredictable state of the application

Reply:
I would say, "Logs that require action from the user" instead of "Should definitely be noted".